src/System.Diagnostics.Debug/src/System.Diagnostics.Debug.builds

```diff
@@ -11,6 +11,14 @@
     <Project Include="System.Diagnostics.Debug.csproj">
       <TargetGroup>net463</TargetGroup>
     </Project>
+    <Project Include="System.Diagnostics.Debug.csproj">
+      <TargetGroup>netcoreapp1.2corert</TargetGroup>
+      <OSGroup>Unix</OSGroup>
```

Comment:
Why did we create a `OSGroup` specific configurations since it seems to me that the Unix and the Windows_NT build for this targetgroup would be exactly the same?